packages/engine-ui/src/assets/thumbs-up.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="88" height="88" viewBox="0 0 88 88" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Icon added twice, both as component and as svg?

Reply:
Forgot to remove after changing implementation! Updated